2.4-taptree.ipynb

```diff
@@ -291,8 +279,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "taptree3 = TapTree()\n",
-    "taptree3.key = pubkey_internal\n",
+    "taptree3 = TapTree(key=pubkey_internal)\n",
```

Comment:
Shall we mention that the huffman constructor only constructs the root? The internal_key still needs to be set by the user.

Reply:
done